pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/segment/FlushThresholdUpdateManager.java

```diff
@@ -78,28 +80,9 @@ public void clearFlushThresholdUpdater(TableConfig tableConfig) {
    */
   private int getLLCRealtimeTableFlushSize(TableConfig tableConfig) {
```

Comment:
Looks like this method is not being used anymore, can we remove it?

Reply:
Yup, removed.